.github/workflows/check.yml

```diff
@@ -71,3 +72,20 @@ jobs:
     - name: Build docs
       if: ${{matrix.build_docs}}
       run: cmake --build '${{github.workspace}}/build' --target doc
+
```

Comment:
We might need a "Build install" step to build stuff for the install artifact... see below.